packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -140,6 +141,10 @@ const seedlessOnboardingMetadata: StateMetadata<SeedlessOnboardingControllerStat
       persist: false,
       anonymous: false,
     },
+    pendingToBeRevokedTokens: {
+      persist: true,
+      anonymous: true,
```

Comment:
its not a already anonymized property right since it has tokens which contains user related info?  All other tokens are marked `anonymous` false so it should be as well.